src/app/frontend/_rtl.scss

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+body:lang(fa) {
+  direction: rtl;
+
+  .mat-form-field {
+    text-align: unset !important;
+  }
+  .mat-paginator-range-actions {
+    flex-direction: row-reverse;
+  }
+  .kd-slider-value {
+    text-align: left !important;
+  }
+  .mat-option {
+    text-align: right;
+    padding: 0 16px !important;
+    padding-right: 32px !important;
+  }
+
+  .mat-optgroup-label {
+    text-align: right;
+  }
+
+  .mat-drawer {
+    transform: translate3d(100%, 0, 0);
+  }
+
+  .mat-slide-toggle {
+    direction: ltr !important;
+  }
+
+  .kd-namespace-select-input-border {
+    text-align: right;
+    input {
+      padding: 16px 16px 16px 0px;
+      text-align: right;
+    }
+  }
+
+  .kd-breadcrumbs-container {
+    mat-icon {
+      transform: rotateZ(180deg);
+    }
+  }
+
+
+
+  ///// Set Font family
+
+  @import url('https://cdn.jsdelivr.net/gh/rastikerdar/vazir-font@v27.0.1/dist/font-face.css');
```

Comment:
We cannot rely on external resources like that. There is also matter of licenses when it comes to using fonts.

Reply:
> We cannot rely on external resources like that. There is also matter of licenses when it comes to using fonts.

I used an external link to do fewer code changes on the project. [Vazir](https://github.com/rastikerdar/vazir-font) is an open-source Persian font and It's very better than Roboto for Persian content. I can add font files manually to avoid using external URLs. But if you think this is not a good idea, I can remove the custom font and use Roboto in Persian, too.

Reply:
If Roboto can be used and there are no specific issues then I'd do this. Otherwise please add font file to the repo.

Reply:
Actually, the issue is only with readability and It’s more beautiful with Vazir font. But now I think it’s better to have a less language-based config to keep the project clean. Because adding the font manually will add about 5MB files to the project and it’s usable only for one language.

Reply:
I removed the custom font.